### PR TITLE
Add ApiKey.getAccount() method

### PR DESCRIPTION
**Pull request by mdeggies**
This adds a getAccount method for the ApiKey resource. It can be used like this:

```
client.getApplication(applicationHref, function (err, application) {
  application.getApiKey("$ID", function(err, apiKey){
    apiKey.getAccount(function(err, account){
      console.log(account);
    });
  });
});
```

I also (finally) wrote unit tests, and plopped them in test/sp.resource.apikey_test.js. Feel free to recommend changes/change stuff- I am happy to refactor!!

### CODE REVIEW

File: lib/resource/ApiKey.js [hunk 1]
```diff
@@ -6,39 +6,38 @@ var InstanceResource = require('./InstanceResource');
 var utils = require('../utils');
 
 /**
- * @class ApiKey
```
Comment (typerandom) [inline lib/resource/ApiKey.js]: You should restore the spaces that you removed from this block comment. For valid JSDoc block comment formatting, please see examples at: http://usejsdoc.org/about-getting-started.html

File: lib/resource/ApiKey.js [hunk 2]
```diff
@@ -84,4 +83,9 @@ ApiKey.prototype._decrypt = function _decrypt(callback) {
   }
 };
 
+ApiKey.prototype.getAccount = function () {
```
Comment (typerandom) [inline lib/resource/ApiKey.js]: This is missing a JSDoc block comment. Please see `AuthenticationResult.getAccount()` as a reference to the comment that you need to add:

```
/**
 * Retrieves the account resource of the user that has been authenticated.
 *
 * @param {ExpansionOptions} [options]
 * For retrieving linked resources of the {@link Account} during this request.
 *
 * @param {Function} Callback
 * Callback function, will be called with (err, {@link Account account}).
 */
```

File: lib/resource/ApiKey.js (hunk 1 above)
Comment (mdeggies) [inline lib/resource/ApiKey.js]: Thanks Robin! Fixed. That was my auto-indenter.

File: lib/resource/ApiKey.js (hunk 2 above)
Comment (mdeggies) [inline lib/resource/ApiKey.js]: Should be good to go. Thanks!

**Comment by typerandom** [comment]
That's awesome. Thank you! :)

**Pull request merged**